crates/shared/src/order_quoting.rs

```diff
@@ -289,12 +300,15 @@ impl TryFrom<QuoteRow> for QuoteData {
 #[mockall::automock]
 #[async_trait::async_trait]
 pub trait OrderQuoting: Send + Sync {
-    /// Computes a quote for the specified order paramters.
+    /// Computes a quote for the specified order paramters. Doesn't store the quote.
     async fn calculate_quote(
         &self,
         parameters: QuoteParameters,
     ) -> Result<Quote, CalculateQuoteError>;
 
+    /// Stores a quote.
+    async fn store_quote(&self, quote: Quote) -> anyhow::Result<Quote>;
```

Comment:
`anyhow::Result` is already in scope and used by the other function signatures. I think it would add unnecessary confusion to use the complete identifier here and in the implementation.

```suggestion
    async fn store_quote(&self, quote: Quote) -> Result<Quote>;
```

Reply:
https://github.com/cowprotocol/services/pull/690/commits/888f0270ee58f72a66eb28bf90640b0699a63e68